scripts/devenv_envs.sh

```diff
@@ -4,7 +4,5 @@ set -euo pipefail
 export OBLIVIUM_CONTAINER_REGISTRY_URL=${OBLIVIUM_CONTAINER_REGISTRY_URL:-"localhost:5000"}
 export OBLIVIUM_CONTAINER_REGISTRY_USERNAME=${OBLIVIUM_CONTAINER_REGISTRY_USERNAME:-}
 export OBLIVIUM_CONTAINER_REGISTRY_PASSWORD=${OBLIVIUM_CONTAINER_REGISTRY_PASSWORD:-}
-export OBLIVIUM_MAVEN_URL=${OBLIVIUM_MAVEN_URL:-"https://software.r3.com/artifactory"}
-export OBLIVIUM_MAVEN_REPOSITORY=${OBLIVIUM_MAVEN_REPOSITORY:-"sgxjvm-maven-nightly"}
 export OBLIVIUM_MAVEN_USERNAME=${OBLIVIUM_MAVEN_USERNAME:-}
 export OBLIVIUM_MAVEN_PASSWORD=${OBLIVIUM_MAVEN_PASSWORD:-}
```

Comment:
I forgot to comment this one before, apologies.
If username and password are only used when publishing, wouldn't be worth to remove those variables as well and have everything under `build.gradle`?